Web/Ecommerce/Ecommerce.Application/ViewModels/Category/CategoryViewModel.cs

```diff
@@ -1,9 +1,11 @@
-﻿namespace Ecommerce.ViewModels.Category
+﻿
+namespace Ecommerce.Application.ViewModels.Category
 {
```

Comment:
convert file scoped namespace all files

Reply:
I don't understand where there is a mistake